### ISSUE TITLE
the error messages in I3ParaboloidFitter::Configure() need improvment (Trac #1510)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1502]
they are all at debug level and use tertiary operator to report "OK" or "PROBLEM" but don't say what is wrong. Use if statements with a log_debug for OK and log_error for Problem and explain the problem.

Migrated from https://code.icecube.wisc.edu/ticket/1510
```json
{
    "status": "closed", 
    "changetime": "2016-01-18T10:53:09", 
    "description": "they are all at debug level and use tertiary operator to report \"OK\" or \"PROBLEM\" but don't say what is wrong. Use if statements with a log_debug for OK and log_error for Problem and explain the problem.", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1453114389228024", 
    "component": "cmake", 
    "summary": "the error messages in I3ParaboloidFitter::Configure() need improvment", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-01-15T18:00:14", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160115T18:00:25: kjmeagher changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160118T10:53:09: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 140981:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140981"
> Fix error messages in paraboloid. now the error messages 1) call log_error if there is a problem 2) explain what the problem is 3) put the error message near the code that determines if there is an error
> fixes #1502
> ```


* kjmeagher changed status from "accepted" to "closed"
